src/Tribe/Views/V2/Widgets/Widget_QR_Code.php

```diff
@@ -227,9 +226,18 @@ public function setup_admin_fields() {
 				'text'  => _x( 'Redirect to the next event in a series', 'Next event in series redirection option', 'the-events-calendar' ),
 			];
 
+			/**
+			 * Filters the series post type.
+			 *
+			 * @since TBD
+			 *
+			 * @param string|null $series_post_type The series post type.
+			 */
+			$series_post_type = apply_filters( 'tec_series_post_type', null );
```

Comment:
No, this is not the way to do this.
The way to do this is to move this all to ECP and put a filter in here for `$options`, letting ECP hook in and add the series options if it is active.